content/legal/release-policy.md

```diff
@@ -301,11 +307,10 @@ licensing FAQ](#license).
 
 #### What are the ASF requirements on approving a release?  {#approving-a-release}
 
-Votes on whether a package is ready to be released use 
-[majority approval](/foundation/glossary.html#MajorityApproval) -- 
-i.e., at least three PMC members must vote affirmatively
-for release, and there must be more positive than negative votes.
-Releases may not be vetoed. Before voting +1 PMC members are required
+Release votes happen as described above in the
+[release approval](#release-approval) section.
+
+Before voting +1 PMC members are required
```

Comment:
The text below is repeated in the previous section. I would suggest removing everything that follows. Alternative we could consider removing the whole section here since it doesn't add anything new to what is already stated before.